cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md

```diff
@@ -0,0 +1,185 @@
+# Troubleshooting Workloads
+
+## <a id="check-build-logs"></a> Check build logs
+
+After a workload is created, users can tail the workload to view the build and runtime logs.
+
+Logs can be checked by running:
+
+```bash
+tanzu apps workload tail pet-clinic --since 10m --timestamp
+ ```
+
+Where:
+
+- `pet-clinic` is the name of the workload.
+- `--since` (optional) the amount of time to go back to begin streaming logs. The default is 1 second.
+- `--timestamp` (optional) prints the timestamp with each log entry.
+
+## <a id="workload-status"></a> Get the workload status and details
+
+After the workload build process is complete, a Knative service can be created to run the workload.
+Workload details can be viewed at any time during the process. Some details, such as the workload
+URL, are only available after the workload is running.
+
+To check the workload details, run:
+
+```bash
+tanzu apps workload get pet-clinic
+```
+
+Where:
+
+`pet-clinic` is the name of the workload to be checked.
+
+Now the workload should be in a running state. When the workload is created, `tanzu apps workload get`
+includes the URL for the running workload. Some terminals allow to `ctrl`+click the URL to
+view it. The URL can also be copy pasted into a web browser to see the application.
+
+## <a id="common-workload-errors"></a> Common workload errors
+
+A workload can either be ready, on error or with an unknown status.
+
+There are known errors that cause the workload to enter an error or unknown status. The most common are:
```

Comment:
There are known errors that cause the workload to enter an error or unknown status. 
Look at the supply chain or delivery steps for status and review the messages section for clues when the workload appears to be having issues.

Below are some of the most common issues with an explanation of the possible cause:

Reply:
updated